NickvisionMoney.Shared/Resources/Strings.nl.resx

```diff
@@ -0,0 +1,677 @@
+	</data>
+
+	<data name="Designers.Credits" xml:space="preserve">
+		<!--DESIGNER NAME(S) GOES HERE. SEPARATE WITH &#xA;-->
+		<value>Nicholas Logozzo https://github.com/nlogozzo&#xA;Fyodor Sobolev https://github.com/fsobolev</value>
+	</data>
+
+	<data name="Artists.Credits" xml:space="preserve">
+		<!--ARTIST NAME(S) GOES HERE. SEPARATE WITH &#xA;-->
+		<value>David Lapshin https://github.com/daudix-UFO</value>
+	</data>
+
+	<data name="Translators.Credits" xml:space="preserve">
+		<!--TRANSLATOR NAME(S) GOES HERE. SEPARATE WITH &#xA;-->
+		<value>Heimen Stoffels https://github.com/Vistaus</value>
+	</data>
+
+	<!--Actions-->
+	<data name="OK" xml:space="preserve">
+		<value>Oké</value>
+	</data>
+
+	<data name="Yes" xml:space="preserve">
+		<value>Ja</value>
+	</data>
+
+	<data name="No" xml:space="preserve">
+		<value>Nee</value>
+	</data>
+
+	<data name="Cancel" xml:space="preserve">
+		<value>Annuleren</value>
+	</data>
+
+	<data name="New" xml:space="preserve">
+		<value>Nieuw</value>
+	</data>
+
+	<data name="Open" xml:space="preserve">
+		<value>Openen</value>
+	</data>
+
+	<data name="Close" xml:space="preserve">
+		<value>Sluiten</value>
+	</data>
+
+	<data name="Save" xml:space="preserve">
+		<value>Opslaan</value>
+	</data>
+
+	<!--Pages-->
+	<data name="Home" xml:space="preserve">
+		<value>Overzicht</value>
+	</data>
+
+	<data name="Account" xml:space="preserve">
+		<value>Rekening</value>
+	</data>
+
+	<data name="Settings" xml:space="preserve">
+		<!--Mostly used in WinUI-->
+		<value>Instellingen</value>
+	</data>
+
+	<data name="Preferences" xml:space="preserve">
+		<!--Mostly used in GNOME-->
+		<value>Voorkeuren</value>
+	</data>
+
+	<data name="KeyboardShortcuts" xml:space="preserve">
+		<value>Sneltoetsen</value>
+	</data>
+
+	<!--MainWindow-->
+	<data name="NickvisionMoneyAccount" xml:space="preserve">
+		<value>Nickvision Money-rekening</value>
+	</data>
+
+	<data name="NewAccount" xml:space="preserve">
+		<value>Nieuwe rekening</value>
+	</data>
+
+	<data name="NewAccount.Tooltip" xml:space="preserve">
+		<value>Nieuwe rekening (Ctrl+N)</value>
+	</data>
+
+	<data name="UnableToOverride" xml:space="preserve">
+		<value>Een rekening kan slechts eenmaal worden geopend.</value>
+	</data>
+
+	<data name="OpenAccount" xml:space="preserve">
+		<value>Rekening tonen</value>
+	</data>
+
+	<data name="OpenAccount.Tooltip" xml:space="preserve">
+		<value>Rekening tonen (Ctrl+O)</value>
+	</data>
+
+	<data name="NoAccountOpened" xml:space="preserve">
+		<value>Er is geen rekening geopend</value>
+	</data>
+
+	<data name="NoAccountDescription" xml:space="preserve">
+		<value>Open een rekening of voeg er een toe om aan de slag te gaan. Ook kunt u een rekeningbestand hierheen slepen.</value>
+	</data>
+
+	<data name="RecentAccounts" xml:space="preserve">
+		<value>Recent Accounts</value>
```

Comment:
Not translated

Reply:
Odd. I guess my text editor didn't save the file after my last edits…